pager/src/main/java/com/google/accompanist/pager/PagerState.kt

```diff
@@ -436,3 +441,21 @@ class PagerState(
         )
     }
 }
+
+/**
+ * A flow which emits the [PagerState.currentPage] if it changes due a scroll
+ * has completed.
+ *
+ * This flow is not meant to be used for updating any UI within the attached [HorizontalPager]
+ * or [VerticalPager]. For that use-case, you should read
+ * [PagerScope.currentPage] and [PagerScope.currentPageOffset] from the content scope.
+ *
+ * @sample com.google.accompanist.sample.pager.PageChangedFlowSample
+ */
+@ExperimentalPagerApi
+inline val PagerState.pageChangedFlow: Flow<Int>
```

Comment:
Don't love encoding the type in the name. What about `pageChanges`? Looking at [`InteractionSource.interactions`](https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/interaction/InteractionSource.kt;l=74) for inspiration.

Reply:
Good call, done.